src/wp-includes/comment.php

```diff
@@ -3232,6 +3234,7 @@ function clean_comment_cache( $ids ) {
 		 */
 		do_action( 'clean_comment_cache', $id );
```

Comment:
this now filres before the caches are deleted, maybe move `wp_cache_delete_multiple` call above this loop?

Reply:
Fixed.